### PR TITLE
branch created and added for homework 6 solution files

### PR DESCRIPTION
**Pull request by Anujagagangras**
please consider this branch for review.

### CODE REVIEW

File: HW6/scripts/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+RUN mkdir /scripts
+RUN mkdir /results
+
+COPY ./Homework_6.sh /scripts/Homework_6.sh
+COPY ./baseball.sql /scripts/baseball.sql
+COPY ./Homework_6.sql /scripts/Homework_6.sql
+RUN chown 1000:1000 /scripts
```
Comment (cmalik3595) [inline HW6/scripts/Dockerfile:9]: Minor question: NOT a COMMENT

Why is the 1000:1000 used here ?  Isn't this already a copy operation from host to container. So container will, by default have the user ownerships of the directories.

File: HW6/scripts/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+RUN apt-get update \
+  && apt-get install --no-install-recommends --yes \
+     mysql-client \
+     wget \
```
Comment (cmalik3595) [inline HW6/scripts/Dockerfile:14]: Minor comment: NOT IMPROTANT

Are you using wget in the code? If not, please remove extra package

File: HW6/scripts/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    container_name: homework_6-rollingaverage
+    depends_on:
+      - mariadb
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./results:/results
+    image: homework_6-rollingaverage
+
```
Comment (cmalik3595) [inline HW6/scripts/docker-compose.yml:27]: Minor comment: CHECK ONLY. 

Container hostname mariadb is not specified in yml file and it is being used in mysql queries on container.

File: HW6/scripts/Homework_6.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+sleep 10
+
+if ! mysql -h mariadb -u root -e 'use baseball'; then
```
Comment (cmalik3595) [inline HW6/scripts/Homework_6.sh:5]: Minor comment:  CHECK ONLY

Please check if hostname = mariadb works here

File: HW6/scripts/Homework_6.sql [hunk 5]
```diff
@@ -0,0 +1,28 @@
+SHOW ERRORS;
```
Comment (cmalik3595) [inline HW6/scripts/Homework_6.sql:1]: JUST SUGGESTION AND THIS COMMENT IS NOT RELATED TO ASSIGNMENT6.

Since this assignment is not specific to validating the rolling average of each batter, I am not looking in the sql file changes. Otherwise "ROWS BETWEEN 99 PRECEDING AND CURRENT ROW" is a bug which we discussed in assignment 2.

**Review by cmalik3595 — COMMENTED** [review]
The code related to Assignment 6 looks clean and bug-free.

However, I have a few comments/suggestions/questions in which we can still optimize the code. Thank you.

File: HW6/scripts/Homework_6.sql (hunk 5 above)
Comment (Anujagagangras) [reply HW6/scripts/Homework_6.sql:1]: Yes.I changed sql code to get actual rolling average according to days and corrected this error of PARTITION by rows. Thanks for the comment.
